OpenRA.Game/FileSystem/ZipFile.cs

```diff
@@ -225,14 +221,13 @@ public static bool TryParseReadWritePackage(string filename, out IReadWritePacka
 				return false;
 			}
 
-			var s = new FileStream(filename, FileMode.Open);
-			package = new ReadWriteZipFile(s, filename);
+			package = new ReadWriteZipFile(filename);
```

Comment:
It should be fine in practice, but it is kind of nasty from a design perspective.  I'll fix this in a followup PR.